python/pyspark/pandas/generic.py

```diff
@@ -3180,6 +3180,7 @@ def __bool__(self) -> NoReturn:
     def _count_expr(spark_column: Column, spark_type: DataType) -> Column:
         # Special handle floating point types because Spark's count treats nan as a valid value,
         # whereas pandas count doesn't include nan.
+        # TODO: Make this work with DataTypeOps.
```

Comment:
Maybe we want to create a related ticket for TODO and comment with ticket number ?

e.g. `TODO(SPARK-xxxxx) Make this work with DataTypeOps.`